### PR TITLE
[X86] Add missing immediate qualifier to the (V)CMPSS/D instructions

### PR DESCRIPTION
**Pull request by RKSimon**
Matches (V)CMPPS/D and makes it easier to algorithmically recreate the instruction name in various analysis scripts I'm working on

### CODE REVIEW

File: llvm/test/CodeGen/X86/domain-reassignment.mir [hunk 1]
```diff
@@ -316,10 +316,6 @@ body:             |
     %11 = VMOVAPDZrrk %2, killed %10, %1
     VMOVAPDZmr %0, 1, $noreg, 0, $noreg, killed %11
 
-    ; FIXME We can't replace TEST with KTEST due to flag differences
-    ; TEST8rr %18, %18, implicit-def $eflags
-    ; JCC_1 %bb.1, 4, implicit $eflags
-    ; JMP_1 %bb.2
```
Comment (goldsteinn) [inline llvm/test/CodeGen/X86/domain-reassignment.mir:322]: Did you mean to drop?

Comment (goldsteinn) [reply llvm/test/CodeGen/X86/domain-reassignment.mir:322]: Likewise below

Comment (RKSimon) [reply llvm/test/CodeGen/X86/domain-reassignment.mir]: No I did not :)

**Review by KanRobert — APPROVED** [review]
LGTM

**Pull request merged**